modules/ownKeys.js

```diff
@@ -0,0 +1,19 @@
+import objectKeys from 'object-keys'
+
+const ownKeys = (object) => {
```

Comment:
@ljharb Not AFAICT. It restricts the result of `Reflect.ownKeys` to enumerable keys.